packages/rest/src/router/routing-table.ts

```diff
@@ -202,7 +208,7 @@ export interface RouteEntry {
   /**
    * OpenAPI operation spec
    */
-  readonly spec: OperationObject;
+  readonly spec: LB4OperationObject;
```

Comment:
This change is not strictly necessary because `OperationObject` type already supports arbitrary extension (additional properties). Having said that, I see how `LB4OperationObject` can be useful because it allows IDEs like VisualStudio Code to offer auto-completion and also the compiler can check that developers don't assign a non-boolean value to this extension by a mistake.

So, if we want to keep `LB4OperationObject`, then please find other places in our public API that are accepting `OperationObject` and change them accordingly. Few examples to get you started:
 - `@operation` decorator and verb-based shortcuts like `@get`
 - `server.route()` method
 - `Route` and `ControllerRoute` classes (and their constructors)

Reply:
Let's keep it and update the references.

Reply:
Replacing `OperationObject ` with `LB4OperationObject ` in all instances is not easy, let's not do it in this PR. So, for not let's keep it as such. Reverting to `readonly spec: OperationObject`.